I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/animalClasses.rpy", line 1, in script
    init python:
  File "game/animalClasses.rpy", line 50, in <module>
    class Trog (Animal):
  File "game/animalClasses.rpy", line 60, in Trog
    determineSex(name)
NameError: name 'name' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/animalClasses.rpy", line 1, in script
    init python:
  File "C:\Users\Shannon\Desktop\renpy-6.99.11-sdk\renpy\ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Shannon\Desktop\renpy-6.99.11-sdk\renpy\python.py", line 1641, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/animalClasses.rpy", line 50, in <module>
    class Trog (Animal):
  File "game/animalClasses.rpy", line 60, in Trog
    determineSex(name)
NameError: name 'name' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.11.1749
 
